File: 백준/Bronze/1159. 농구 경기/농구 경기.cc
#include <bits/stdc++.h>
using namespace std;

string temp, ans;
int arr[26];
int n, i;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);
	
	cin >> n;

	for (i = 0; i < n; ++i) {
		cin >> temp;
		arr[temp[0] - 'a']++;
	}

	for (i = 0; i < 26; ++i) {
		if (arr[i] >= 5)	ans += (i + 'a');
	}

	if (ans.length() > 0) {
		cout << ans;
	}
	else {
		cout << "PREDAJA";
	}

	return 0;
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.py
import sys
input = sys.stdin.readline

n = int(input())

hash = dict()

for _ in range(n):
    name = input().strip()

    if name[0] in hash:
        hash[name[0]] += 1
    else:
        hash[name[0]] = 1

arr = list(hash.items())

arr.sort(key = lambda x : (-x[1], x[0]))

ans = []

for k, v in arr:
    if v >= 5:
        ans.append(k)

ans.sort()

if ans:
    print("".join(ans))
else:
    print("PREDAJA")